### DISCUSSION TITLE
[FEATURE]本地应用由于配置错误导致安装失败后，支持在应用商店中删除该应用的记录。

### DISCUSSION THREAD

**Discussion by IshuaWang** [1Panel-dev/1Panel#6607] [Q&A]
### 1Panel 版本

v1.10.15-lts

### 问题描述

我尝试本地安装自己的应用，可能是由于配置的有问题，导致安装失败了。然后应用商店就出现了下面的这个项目，也没有办法删除。我把本地文件夹里的文件删除了，这个东西还是存在，不知道怎么去掉。
![QQ20240928-093326](https://github.com/user-attachments/assets/2cc6f11f-9c88-4afa-9010-26cfe2229bc7)

### 重现步骤

我就是在本地商店新建了个应用的文件夹，然后按照官方文档那样去配置，但是配置的可能有问题。

### 期待的正确结果

就是安装失败应该可以在应用商店把对应的应用删掉。

### 相关日志输出

```shell
time="2024-09-28T09:39:37+08:00" level=warning msg="The \"ENV_FILE\" variable is not set. Defaulting to a blank string."
Failed to load /opt/1panel/apps/local/outline/localoutline2: read /opt/1panel/apps/local/outline/localoutline2: is a directory
```

**Comment by wanghe-fit2cloud** [comment]
进入 `工具箱` 页面，点击右上角的 `重启面板` 按钮，或者手动执行 `1pctl restart` 命令重启面板，是否可以在 `已安装` 页面勾选强制删除应用记录？

**Reply by IshuaWang** [reply]
重启之后可以删除了
